pyproject.toml

```diff
@@ -1,60 +1,192 @@
-  "PC180",  # Uses a markdown formatter
-  # https://learn.scientific-python.org/development/guides/style/#PC190
-  "PC190",  # Uses Ruff
-  # https://learn.scientific-python.org/development/guides/style/#PC901
-  "PC901",  # Custom pre-commit CI message
-
-  # https://learn.scientific-python.org/development/guides/style/#MY100
-  "MY100",  # Uses MyPy (pyproject config)
-
-  # https://learn.scientific-python.org/development/guides/style/#RF001
-  "RF001",  # Has Ruff config
+    # https://learn.scientific-python.org/development/guides/style/#PC180
+    "PC180",  # Uses prettier
+
+    # https://learn.scientific-python.org/development/guides/packaging-simple/#PY005
+    "PY005",  # Has tests folder
+
+    # TODO: exceptions that still need investigating are below. Might be fixable, or might become permanent (above):
+
+    # https://learn.scientific-python.org/development/guides/pytest/#PP301
+    "PP301", # Has pytest in pyproject
+
+    # https://learn.scientific-python.org/development/guides/gha-basic/#GH212
+    "GH212",  # Require GHA update grouping
+
+    # https://learn.scientific-python.org/development/guides/style/#PC170
+    "PC170",  # Uses PyGrep hooks (only needed if rST present)
+
+    # https://learn.scientific-python.org/development/guides/style/#PC901
+    "PC901",  # Custom pre-commit CI message
+]
+
+
+[tool.mypy]
+strict = false
+ignore_missing_imports = true
+warn_unused_configs = true
+warn_unreachable = true
+enable_error_code = ["ignore-without-code", "redundant-expr", "truthy-bool"]
+exclude = [
+
+]
+
+[tool.numpydoc_validation]
+checks = [
+    "all",  # Enable all numpydoc validation rules, apart from the following:
+
+    # -> Docstring text (summary) should start in the line immediately
+    #    after the opening quotes (not in the same line, or leaving a
+    #    blank line in between)
+    "GL01",  # Permit summary line on same line as docstring opening quotes.
+
+    # -> Closing quotes should be placed in the line after the last text
+    #    in the docstring (do not close the quotes in the same line as
+    #    the text, or leave a blank line between the last text and the
+    #    quotes)
+    "GL02",  # Permit a blank line before docstring closing quotes.
+
+    # -> Double line break found; please use only one blank line to
+    #    separate sections or paragraphs, and do not leave blank lines
+    #    at the end of docstrings
+    "GL03",  # Ignoring.
+
+    # -> See Also section not found
+    "SA01",  # Not all docstrings require a "See Also" section.
+
+    # -> No extended summary found
+    "ES01",  # Not all docstrings require an "Extended Summary" section.
+
+    # -> No examples section found
+    "EX01",  # Not all docstrings require an "Examples" section.
+
+    # -> No Yields section found
+    "YD01",  # Not all docstrings require a "Yields" section.
+
+    # TODO: exceptions that still need investigating are below. Might be fixable, or might become permanent (above):
```

Comment:
This line can probably be removed now.

Reply:
https://github.com/SciTools/tephi/pull/172/commits/c0eeb8066621debbf4aa3444c3ddc7ce153179e9